### DISCUSSION TITLE
Config Confusion

### DISCUSSION THREAD

**Discussion by rgc1210** [meisnate12/Plex-Meta-Manager#1707] [Q&A]
Hi
I've just found this PMM and have installed it Ok and got it working with the default config.yml but I'm confused as to how to implemt the fantastic overlays onto posters that I see under the "Example Poster Overlay" which I believe are defaults. Can anyone help me understand if I am to change the default config.yml to get these overlays  or do I add another file called overlays.yml (with the relevant details) into the config folder? and then add a reference to it in the config.yml. I tried using  this method but got a error "YAML overlays attribute not configured."
Help much appreciated

**Comment by bullmoose20** [comment]
> Hi
> 
> I've just found this PMM and have installed it Ok and got it working with the default config.yml but I'm confused as to how to implemt the fantastic overlays onto posters that I see under the "Example Poster Overlay" which I believe are defaults. Can anyone help me understand if I am to change the default config.yml to get these overlays  or do I add another file called overlays.yml (with the relevant details) into the config folder? and then add a reference to it in the config.yml. I tried using  this method but got a error "YAML overlays attribute not configured."
> 
> Help much appreciated 

If you have gone through the full walk through, on our wiki, you will find all you need and it even applies overlays as part of that walk through. You will really benefit from something like this. You can also join our discord server to ask questions for further clarity.
